### PR TITLE
[Deployment] Ignite K8s yaml, docker, and xml files

### PR DESCRIPTION
**Pull request by xieus**
This PR adds new configuration files for Ignite
- K8s yaml file
- xml file customized for Alcor
- Docker file

### CODE REVIEW

**Comment by xieus** [comment]
@Gzure This is current version of Ignite deployment files.

File: kubernetes/db/ignite/ignite_alcor.yaml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ignite-alcor
+  namespace: ignite-alcor
+---
+apiVersion: v1
+kind: Service
+metadata:
+  # The name must be equal to TcpDiscoveryKubernetesIpFinder.serviceName
```
Comment (er1cthe0ne) [inline kubernetes/db/ignite/ignite_alcor.yaml:44]: may remove this comment since it is not applicable.

**Comment by Gzure** [comment]
@kevin-zhonghao
Ignite docker build file
 `# Dockerfile for Customized Ignite Image

FROM apacheignite/ignite:2.8.1

MAINTAINER Liguang Xie <lxie@futurewei.com>

# Generate container image and run container
COPY ./target/common-0.1.0-SNAPSHOT.jar /opt/ignite/apache-ignite/libs/common-0.1.0-SNAPSHOT.jar

CMD $IGNITE_HOME/run.sh

# Container port exposure
EXPOSE 11211 47100 47500 49112 10800 8080
`
Ignite deployment yaml
`apiVersion: apps/v1
kind: Deployment
metadata:
  name: ignite-cluster
spec:
  selector:
    matchLabels:
      app: ignite
  replicas: 3
  template:
    metadata:
      labels:
        app: ignite
    spec:
      #hostNetwork: true
      serviceAccountName: ignite
      containers:
      - name: ignite-node
        image: ignite:v1.2
        #image: apacheignite/ignite:2.8.1
        imagePullPolicy: IfNotPresent
        resources:
          requests:
            cpu: "16"
        env:
        - name: OPTION_LIBS
          value: ignite-kubernetes,ignite-rest-http
        - name: CONFIG_URI
          value: /opt/ignite/config/ignite-config.xml
        - name: IGNITE_QUIET
          value: "false"
        - name: JVM_OPTS
          value: "-Djava.net.preferIPv4Stack=true"
        ports:
        - containerPort: 11211 # REST port number.
        - containerPort: 47100 # communication SPI port number.
        - containerPort: 47500 # discovery SPI port number.
        - containerPort: 49112 # JMX port number.
        - containerPort: 10800 # SQL port number.
        - containerPort: 10900 # Thin clients port number.
        volumeMounts:
        - name: ignite-config-volume
          mountPath: /opt/ignite/config
      volumes:
        - name: ignite-config-volume
          configMap:
            name: ignite-config`

**Comment by xieus** [comment]
@kevin-zhonghao Could you please arrange to check in the latest yaml files to the master? Either update this PR or start a new PR.

**Pull request closed without merging**